src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOperatorBase.java

```diff
@@ -0,0 +1,16 @@
+package se.liu.ida.hefquin.engine.queryplan.logical.impl;
+import se.liu.ida.hefquin.engine.queryplan.logical.LogicalOperator;
```

Comment:
By convention, in Java code it is typical to always have a single empty line between the package and the first import statement.

```suggestion
package se.liu.ida.hefquin.engine.queryplan.logical.impl;

import se.liu.ida.hefquin.engine.queryplan.logical.LogicalOperator;
```